lib/bank_accounts/account.rb

```diff
@@ -0,0 +1,45 @@
+require 'pry'
+
+module BankAccounts
+  class Account
+
+    attr_reader :id, :balance
+
+    # Rounds float value down to hundredths position
+    # For example 10.2599 rounds to 10.25
+    def round_to_hundredths(n)
+      n = (((n * 100).floor) / 100.0)
+      return n
+    end
```

Comment:
I like your round_to_hundredths implementation. You have clearly considered the context of your program.